src/puppetlabs/trapperkeeper/authorization/acl.clj

```diff
@@ -11,7 +13,8 @@
 (def Extensions
   "Schema for representing SSL Extensions. Maps from a keyword shortname to a
   string value."
-  {schema/Keyword schema/Str})
+  {schema/Keyword schema/Str
+   (schema/optional-key :subjectAlternativeName) [schema/Str]})
```

Comment:
We'd talked about this already, but I think it would be good to get a consult from someone on the docs team about what we want to call this exactly.  Where the other standard Puppet ones would have names like `pp_uuid`, maybe something like `subject_alternative_name` would be better for consistency?